wwwroot/js/site.js

```diff
@@ -8,12 +8,12 @@ function getWords() {
         .catch(error => console.error('Unable to show words.', error));
 }
 
-function getWordOfTheDay() {
-    fetch(uri)
-        .then(response => response.json())
-        .then(data => _displayWordOfTheDay(data))
-        .catch(error => console.error('Unable to show words.', error));
-}
+//function getWordOfTheDay() {
```

Comment:
No need to comment code. Just remove it if you don't need it. Git history contains all your changes, so you can revert any changes in the future if you need.